src/auth.c

```diff
@@ -1716,13 +1716,16 @@ int pr_auth_chroot(const char *path) {
 
   tz = pr_env_get(session.pool, "TZ"); 
   if (tz == NULL) {
+    char tzbuf[32];
+    long off = (timezone / 60 / 60) - daylight;
+    snprintf(tzbuf, sizeof(tzbuf), "%s%ld", daylight ? tzname[1] : tzname[0], off);
```

Comment:
I'm trying to address the issue you described at http://www.proftpd.org/docs/howto/Timestamps.html, but without requiring the user to do any special configuration.

I figured the existing GLIBC ifdef's would be enough to weed out platforms that wouldn't work with this code, but certainly that assumption could be invalid.